### DISCUSSION TITLE
zh/datasets/pose/hand-keypoints/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#18257] [Docs]
# zh/datasets/pose/hand-keypoints/

探索用于高级姿势估计的手部关键点估计数据集。了解使用YOLO 进行训练的数据集、预训练模型、指标和应用。

https://docs.ultralytics.com/zh/datasets/pose/hand-keypoints/

**Comment by berlintop** [comment]
I used yolov11n and yolov11x to train many models based on this dataset. The log accuracy of small batch and large epoch training is very good, but it is very poor during actual inference, much worse than mediaPipa, especially when rotating, the key points are very inaccurate. I am very confused about this, is there any trick to the training parameters?

**Reply by glenn-jocher** [reply]
@737296 the issue you described could be due to overfitting or suboptimal augmentation and training practices. To address inaccuracies during inference, consider increasing data augmentation (e.g., rotation, scaling), fine-tuning learning rates or epochs, and ensuring sufficient diversity in the dataset. MediaPipe's robustness might stem from its architecture and preprocessing, so refining training parameters or exploring a different YOLO11 variant could help. Refer to the [Hand Keypoints Dataset](https://docs.ultralytics.com/datasets/pose/hand-keypoints/) documentation for guidance on dataset setup and training. Let us know if further clarification is needed!

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for your interest in Ultralytics 🚀! It’s great to see your exploration of hand keypoints estimation datasets for advanced pose estimation using YOLO. We recommend visiting the [Docs](https://docs.ultralytics.com/zh/datasets/pose/hand-keypoints/) for an in-depth understanding of this topic, including datasets, pretrained models, metrics, and potential applications.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us debug it.

If this is a custom training ❓ Question, please include detailed information such as example data, training configurations, and any relevant logs. Make sure to follow our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) for optimal performance.

Join the Ultralytics community where it suits you best:
- For real-time discussion, connect on [Discord](https://discord.com/invite/ultralytics) 🎧
- Share or browse insights on [Discourse](https://community.ultralytics.com)
- Engage with our community on [Reddit](https://reddit.com/r/Ultralytics)

## Upgrade

Ensure you’re using the latest version of `ultralytics` with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) for best results:

```bash
pip install -U ultralytics
```

## Environments

Run YOLO in these verified environments with all dependencies preinstalled:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response, but don’t worry! An Ultralytics engineer will be here shortly to assist you further 😊.

**Comment by MengBo-GitHub** [comment]
感谢如此详细的教学文档，可惜在个人学习使用阶段没有足够的 训练条件 ，我请求分享一些训练后的模型提供给学习者使用，感激不尽😄

**Reply by glenn-jocher** [reply]
感谢支持！目前我们未提供基于 Hand Keypoints 的官方预训练权重；建议从 YOLO11n‑pose.pt 做迁移学习，或使用 Ultralytics HUB 进行一键云端训练获取可用模型，如有社区训练的 checkpoint 也欢迎在此贴共享链接。
